### ISSUE TITLE
Image alt attributes

### ISSUE THREAD

**Issue by stevetoro** [esunshin/prj-rev-bwfs-tea-cozy#4]
https://github.com/esunshin/prj-rev-bwfs-tea-cozy/blob/master/Unit%20Final%20Project/index.html#L31

Try to give all of your `img` tags `alt` attributes that describe the image.

This helps improve the accessibility of your site when it's unable to render images, like when visually impaired users use readers that don't display images.
